dashboard/test/factories/factories.rb

```diff
@@ -150,6 +150,58 @@
           end
         end
       end
+      trait :with_google_authentication_option do
```

Comment:
Is it necessary to duplicate these traits inside `teacher` when they're already defined in `user`?

Reply:
oops, nevermind, i just lost track of nesting in the factory! i'll remove the extraneous factory from `teacher`